src/Header/Header.jsx

```diff
@@ -115,6 +115,21 @@ function Header() {
     return (
       <>
         <aside
+          className={cx("mobile-nav-background-simple", {
+            visible: mobileNavBackgroundIsShowing,
+            hidden: !mobileNavBackgroundIsShowing,
+          })}
+        />
+        <button
+          className={cx("mobile-nav-background-close-button-simple", {
+            visible: mobileNavBackgroundIsShowing,
+            hidden: !mobileNavBackgroundIsShowing,
+          })}
+          onClick={hideMobileNav}
+        >
+          <FontAwesomeIcon icon="times" className="icon" color="#fefefe" />
+        </button>
+        {/* <aside
```

Comment:
These will be uncommented when we go back to the proper header.

Reply:
Just a thought for when we return to the better version (where we have `mobile-nav-background-1`, `mobile-nav-background-2` and `mobile-nav-background-3`), it would be nice if we could have them all hidden away in a component, so that the code is simple like what we have here. What do you think?

Reply:
Yeah, that's a good idea. We should make this more robust. At the very least, the Header component should instantiate a MobileHeader and a DesktopHeader inside it, and toggle accordingly.